src/uu/truncate/src/truncate.rs

```diff
@@ -115,7 +115,16 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches = uu_app()
         .override_usage(&usage[..])
         .after_help(&long_usage[..])
-        .get_matches_from(args);
+        .try_get_matches_from(args)
+        .unwrap_or_else(|e| {
+            e.print().expect("Error writing clap::Error");
+            match e.kind {
+                clap::ErrorKind::DisplayHelp | clap::ErrorKind::DisplayVersion => {
+                    std::process::exit(0)
```

Comment:
Nit to make it a bit cleaner using `UResult` and so that we don't have to call `std::process:exit`:
```rust
.map_err(|e| {
    e.print().expect("Error writing clap::Error");
    match e.kind {
        clap::ErrorKind::DisplayHelp | clap::ErrorKind::DisplayVersion => 0,
        _ => 1,
    }
})?;
```

Reply:
I love it. Working on it now.